Extensions/dist/page/SafariUpdate.plist

```diff
@@ -10,11 +10,11 @@
        <key>Developer Identifier</key>
        <string>JTRG49NC7Z</string>
        <key>CFBundleVersion</key>
-       <string>7.8.2</string>
+       <string>7.9.0</string>
        <key>CFBundleShortVersionString</key>
-       <string>7.8.2</string>
+       <string>7.9.0</string>
        <key>URL</key>
-       <string>https://github.com/new-xkit/XKit/releases/download/v7.8.2/New.XKit.safariextz</string>
+       <string>https://github.com/new-xkit/XKit/releases/download/v7.9.0/New.XKit.safariextz</string>
```

Comment:
I'll update the name accordingly

Reply:
beat you to it before i opened the PR :P

Reply:
Oh it was already updated, nice!